storage/azure/src/server/AzureServerStorage.ts

```diff
@@ -212,6 +215,7 @@ export class AzureServerStorage extends ServerStorage {
       reference,
       size: contentLength!,
       metadata,
+      contentEncoding: _response.parsedHeaders.contentEncoding,
```

Comment:
Are both `_response` and `parsedHeaders` always guaranteed to be not null? maybe it's worth to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here

Reply:
they are always defined, unless the type definitions are wrong